### DISCUSSION TITLE
Is it possible to send/recieve node ipc messages to/from spawned processes?

### DISCUSSION THREAD

**Discussion by mrfigg** [Microsoft/node-pty#615] [Q&A]
## Environment details

- OS: Raspberry Pi OS Lite (64-bit) Debian GNU/Linux 11 (bullseye)
- node-pty version: 0.10.1

## Issue description
Basically I want to reproduce [`child_process.spawn()` ipc channels](https://stackoverflow.com/a/41037698) with node-pty. Is this possible?

**Comment by Tyriar** [comment]
No, you need to use the onData listener to listen to the output.
